metaci/build/models.py

```diff
@@ -713,6 +713,19 @@ def _get_flow_options(self) -> dict:
                 "sandbox_date": self.build.release.sandbox_push_date,
                 "production_date": self.build.release.production_push_date,
             }
+        if (
+            self.build.plan.role == "push_sandbox" and self.build.release
+        ):  # override lives in MetaCI
+            options["push_sandbox"] = {
+                "version": f"{self.build.release.version_number}",
```

Comment:
This field isn't currently required, should I just parse the git tag? Or should I make a migration to make this required? @davisagli

Reply:
We should probably make it required.

I think we could also potentially set defaults for both git tag and version number based on the most recent beta tag, but that should probably be a new work item and separate PR